frontend/src/scenes/surveys/SurveyCustomization.tsx

```diff
@@ -99,17 +99,23 @@ export function Customization({ appearance, surveyQuestionItem, onAppearanceChan
                     onChange={(submitButtonColor) => onAppearanceChange({ ...appearance, submitButtonColor })}
                     disabled={!surveysStylingAvailable}
                 />
+                <div className="mt-2">Button text color</div>
+                <LemonInput
+                    value={appearance?.submitButtonTextColor}
+                    onChange={(submitButtonTextColor) => onAppearanceChange({ ...appearance, submitButtonTextColor })}
+                    disabled={!surveysStylingAvailable}
+                />
                 {surveyQuestionItem.type === SurveyQuestionType.Open && (
                     <>
-                        <div className="mt-2">Placeholder</div>
+                        <div className="mt-2">Placeholder text</div>
                         <LemonInput
                             value={appearance?.placeholder || defaultSurveyAppearance.placeholder}
                             onChange={(placeholder) => onAppearanceChange({ ...appearance, placeholder })}
                             disabled={!surveysStylingAvailable}
                         />
                     </>
                 )}
-                <div className="mt-2">
+                <div className="mt-4">
```

Comment:
noticed that the customization elements seemed too crowded; added a little space here to let the different parts breathe.

Before: 
<img width="478" alt="image" src="https://github.com/user-attachments/assets/8be9d3b3-f0c5-4431-b87b-d6ee3413747a">

After: 
<img width="485" alt="image" src="https://github.com/user-attachments/assets/6526f9f0-be92-4f86-9855-3dd8afbf186f">